bellows/zigbee/application.py

```diff
@@ -49,6 +49,8 @@ def initialize(self):
         yield from self._cfg(c.CONFIG_APPLICATION_ZDO_FLAGS, zdo)
         yield from self._cfg(c.CONFIG_TRUST_CENTER_ADDRESS_CACHE_SIZE, 2)
         yield from self._cfg(c.CONFIG_PACKET_BUFFER_COUNT, 0xff)
+        yield from self._cfg(c.CONFIG_KEY_TABLE_SIZE, 1)
+        yield from self._cfg(c.CONFIG_TRANSIENT_KEY_TIMEOUT_S, 180)
```

Comment:
My HUSBZB-1 doesn't seem to support CONFIG_TRANSIENT_KEY_TIMEOUT_S   (EzspStatus.ERROR_INVALID_ID)

The device reports as EZSP version 4. I'm not aware of any way to do a firmware update?